github-management/README.md

```diff
@@ -56,9 +56,9 @@ GitHub organization.
 They also have approval privileges for adding new members to the GitHub config.
 
 Our current coordinators are:
-* Naeil Ezzoueidi (**[@nzoueidi](https://github.com/nzoueidi)**, Central European)
-* Stephen Augustus (**[@justaugustus](https://github.com/justaugustus)**, US Eastern)
-* Yang Li (**[@idealhack](https://github.com/idealhack)**, Japan Standard Time)
+* Arnaud Meukam (**[@ameukam](https://github.com/ameukam)**, Central European)
+* Nabarun Pal (**[@palnabarun](https://github.com/palnabarun)**, Indian Standard Time)
+* Savitha Raghunathan (**[@idealhack](https://github.com/savitharaghunathan)**, US Eastern)
```

Comment:
```suggestion
* Savitha Raghunathan (**[@savitharaghunathan](https://github.com/savitharaghunathan)**, US Eastern)
```

Reply:
This is what you get while trying to create PRs between meetings :woman_facepalming: 

Fixed :+1: